src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2253,6 +2253,103 @@ public function get_test_authorization_header() {
 		return $result;
 	}
 
+	/**
+	 * Tests persistent_object_cache health check.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @return array The health check result suggesting persistent object caching.
```

Comment:
It won't necessarily suggest persistent object caching - it might also say that everything is fine. I would suggest we rephrase it to the same description that is used in other core methods for Site Health tests.
```suggestion
	 * @return array The test result.
```